lib/commands/general.js

```diff
@@ -92,11 +103,24 @@ export async function removeApp(appId) {
 /**
  * @this RokuDriver
  * @param {string} appId
+ * @param {string} [contentId]
+ * @param {string} [mediaType]
  */
-export async function activateApp(appId) {
+export async function activateApp(appId, contentId, mediaType) {
   this._cachedSourceDirty = true;
   log.info(`Launching app ${appId}`);
-  await this.rokuEcp(`/launch/${appId}`);
+  let launchUrl = `/launch/${appId}`;
+  if (contentId) {
+    if (!mediaType) {
+      throw new Error(`If you include a contentId parameter to activate, must also include mediaType`);
+    }
+    if (!VALID_MEDIA_TYPES.has(/** @type ValidMediaTypes */(mediaType))) {
```

Comment:
This tag either actually works or it's simply invalid (which defaults to the `any` type, which would also work).

```suggestion
    if (!VALID_MEDIA_TYPES.has(/** @type {ValidMediaTypes} */(mediaType))) {
```